modules/imap/output_modules.php

```diff
@@ -376,9 +376,6 @@ protected function output() {
             if ($msg_part === '0') {
                 $txt .= ' | <a class="normal_link hlink msg_part_link normal_link" data-message-part="" href="#">'.$this->trans('normal').'</a>';
```

Comment:
We don't have any use for this anymore. Please remove it as well!

Reply:
it's done